misc/gas_sensor/gas_sensor.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component version="1.4" xmlns="http://schemas.circuit-diagram.org/circuitDiagramDocument/2012/component/xml">
+  <declaration>
+    <meta name="name" value="Gas Sensor" />
+    <meta name="minsize" value="50" />
+    <meta name="author" value="chauhannaman98" />
+    <meta name="guid" value="8f764035-18c6-4421-ac7f-5cbd327d4e78" />
+
+    </declaration>
+
+  <connections>
+    <group>
+      <connection name="VCC" start="_Start-30y" end="_Middle-10x-30y" edge="start" />
+      <connection name="GND" start="_Start-10y" end="_Middle-10x-10y" edge="start" />
+      <connection name="A0" start="_Start+10y" end="_Middle-10x+10y" edge="start" />
+      <connection name="D0" start="_Start+30y" end="_Middle-10x+30y" edge="start" />
```

Comment:
```suggestion
      <connection name="VCC" start="_Start-30y" end="_End-50x-30y" edge="start" />
      <connection name="GND" start="_Start-10y" end="_End-50x-10y" edge="start" />
      <connection name="A0" start="_Start+10y" end="_End-50x+10y" edge="start" />
      <connection name="D0" start="_Start+30y" end="_End-50x+30y" edge="start" />
```
align to end instead of middle - fixes resizing